src/script/main/app.ts

```diff
@@ -388,7 +418,7 @@ class App {
         properties: propertiesRepository,
         team: teamRepository,
         user: userRepository,
-      } = this.repository;
```

Comment:
The type declaration shouldn't be needed since `this.repository` is typed.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/9511/commits/f89353a3a5793eaa4bf6a95c1cd629ec63828ee1